src/lucky/secure_headers/set_frame_guard.cr

```diff
@@ -0,0 +1,56 @@
+module Lucky
+  module SecureHeaders
+    # This module sets the HTTP header [X-Frame-Options](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Frame-Options).
+    # It's job is responsible for deciding which site can call your site from within a frame.
+    # For more information, read up on [Clickjacking](https://en.wikipedia.org/wiki/Clickjacking).
+    #
+    # Include this module in the actions you want to add this to.
+    # A required method `frame_guard_options` must be defined`
+    # ```
+    # class BrowserAction < Lucky::Action
+    #   include Lucky::SecureHeaders::SetFrameGuard
+    #
+    #   def frame_guard_options
+    #     {allow_from: "deny"}
+    #   end
+    # end
+    # ```
+    #
+    # ### Options
+    # The `frame_guard_options` method must be defined and return `NamedTuple(allow_from: String)`
+    # The `allow_from` key can have one of 3 String values:
+    # - `"same"` or `"sameorigin"`
+    # - `"nowhere" or `"deny"`
```

Comment:
I believe these lines need to be updated

Reply:
yup. Good call.